config/application.yml.example

```diff
@@ -30,6 +30,7 @@ SMTP_USERNAME: 'ofn'
 SMTP_PASSWORD: 'f00d'
 
 # Optional mail settings
+# MAIL_SECURE_CONNECTION: 'TLS' # 'None' (default), 'SSL' or 'TLS'
```

Comment:
I don't think there's ever a reason to use SSL. Should be TLS only.

Reply:
I'm not sure about the naming here. TLS could mean STARTTLS on port 587. That is the configuration we use at the moment. And SSL could mean TLS on port 465. Both are secure connections as long as the underlying TLS implementation is secure. https://www.fastmail.com/help/technical/ssltlsstarttls.html

And it looks like TLS over port 465 was nominated the winner this year: https://tools.ietf.org/html/rfc8314#section-3

Maybe we should derive the connection method from the port? It's very unlikely that somebody wants to use the wrong encryption on the wrong port.

Reply:
I am not sure but after reading a few things including your links, I think you may be right and "SSL could mean TLS on port 465".